bitnami/clickhouse/templates/statefulset.yaml

```diff
@@ -25,8 +25,10 @@ spec:
   {{- end }}
   template:
     metadata:
-      {{- if or $.Values.podAnnotations (and $.Values.metrics.enabled $.Values.metrics.podAnnotations) }}
       annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") $ | sha256sum }}
+        checksum/config-extra: {{ include (print $.Template.BasePath "/configmap-extra.yaml") $ | sha256sum }}
+        {{- if or $.Values.podAnnotations (and $.Values.metrics.enabled $.Values.metrics.podAnnotations) }}
```

Comment:
You don't need this:
```
        {{- if or $.Values.podAnnotations (and $.Values.metrics.enabled $.Values.metrics.podAnnotations) }}
```
That condition was included at the beginning of the block to avoid an empty `annotations:` in the yaml. In this case we will have always at least the checksum, so you don't need the condition. Please remove also the final `{{- end }}` for that `if`

Reply:
That's done, please have a look